src/components/RoundForm.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <div>
+    <h2>Input Rounds</h2>
+    <p>{{ msg }}</p>
+    
+    <input v-model="date" type="text" size="35" placeholder="Enter date, yyyy-mm-dd"><br><br>
+    
+    <select v-model="club" @change="changeClub">
+      <option :value="{}" default selected disabled>Select club</option>
+      <option v-for="clubOpt in clubOptions" :key="clubOpt.name"
+      :value="clubOpt">
+        {{ clubOpt.name }}
+      </option>
+    </select><br><br>
+
+    <table>
+      <tr v-for="(course, i) in courses" :key="course.name">
+        <td>{{ course.name }}</td>
+        <td>
+          <button @click="removeCourse(i)">Remove</button>
+        </td>
+      </tr>
+      <tr v-show="canAddCourses">
+        <td>
+          <select v-model="newCourse">
+            <option :value="{}" default selected disabled>Select course</option>
+            <option v-for="course in newCourseOptions" :key="course.name"
+            :value="course">
+              {{ course.name }}
+            </option>
+          </select>
+        </td>
+        <td>
+          <button @click="addCourse"
+          :disabled="Object.keys(newCourse).length == 0">
```

Comment:
`Object.keys...` is messy, just want to check if a course has been selected yet, ie. it's not equal to the default value `{}`. Could use a framework like Lodash or Underscore to clean it up to `_.isEmpty(newCourse)` but that's overkill right now since there's no other reason to use those frameworks